lib/expectations.js

```diff
@@ -31,9 +31,6 @@ function normalizeMetrics(metrics) {
 }
 
 function checkExpectations(metricsData, expectationMetrics) {
-  validateMetrics(expectationMetrics);
```

Comment:
We can now remove this since it being called right off the bat in our constructor.